src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/outertypefilename/Example2.java

```diff
@@ -0,0 +1,18 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="OuterTypeFilename"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.outertypefilename;
+
+// xdoc section -- start
+// public class Example1 {
+// // This class is commented out
+// }
```

Comment:
please remove such commented out code.
we have description over example like `Example of class Foo2 in a file named Example2.java` so it is enough to understand an problem for user.

Reply:
I added the commented class to satisfy the `exampleClassName` check.
if we remove those, we will have to add suppressions.

Should I proceed with adding new suppressions ?

Reply:
Yes suppression, as this module by design needs different name.
Or you can try make other class to be name Example1ButNotSameName it might pass validation:)

Reply:
> Or you can try make other class to be name `Example1ButNotSameName `it might pass validation:)

Cheat-code applied successfully  ;)